specs/Autofill.md

```diff
@@ -0,0 +1,107 @@
+# Background
+The WebView2 team has been asked for an API to allow end developers to set the autofill preferences.  We are exposing two of the autofill preferences that will allow enabling and disabling general and password autofill.  General autofill includes things like email addresses, shipping addresses, phone numbers, and names.  Password autofill includes things like usernames and passwords for login.  Password information is not included in the general autofill. 
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+
+Autofill has three components
+* Auto-populate - Populate the corresponding form fields automatically on page load.
+* Suggest - When the user clicks on the form field, drop down suggestions of previously saved forms will be displayed.
+* Populate - When clicking on one of the suggestions, the form data will populate the respective fields.
+The general autofill setting and password autofill setting behave independently.  Their behavior differs as well.  
+When the password autofill setting is disabled, no new password data is saved and no Save/Update Password prompts are displayed.  However, if there was password data already saved before disabling this setting, then that password information is auto-populated, suggestions are shown and clicking on one will populate the fields.  When the password autofill setting is enabled, password information is auto-populated, suggestions are shown and clicking on one will populate the fields, new data is saved, and a Save/Update Password prompt is displayed. The password autofill setting default behavior is enabled. 
+When the general autofill setting is disabled, no suggestions appear, and no new information is saved. When the general autofill setting is enabled, information is saved, suggestions appear and clicking on one will populate the form fields. 
+
+
+# Examples
+
+## Win32 C++
+```cpp
+void SettingsComponent::TogglePasswordAutofill() {
+    wil::com_ptr<ICoreWebView2Settings4> settings;
+    webView->get_Settings(&settings);
```

Comment:
get_Settings gives out an ICoreWebView2Settings not an ICoreWebView2Settings4. You'll need two com_ptrs one for collecting the ICoreWebView2Settings from get_Settings and another to cast to to call get/put_IsPass...